src/material.cpp

```diff
@@ -36,8 +36,8 @@ material_type::material_type()
 {
     _ident = material_id( "null" );
     _name = "null";
-    _salvage_id = "null";
-    _salvage_multiplier = 1.0;
+    _salvaged_into = itype_id( "null" );
+    _repaired_with = itype_id( "" );
```

Comment:
In this PR or next? I can wait and test the others PRs first if you want to push this here.